infra/build.gradle.kts

```diff
@@ -80,7 +80,7 @@ dependencies {
     implementation("net.postgis:postgis-jdbc:2021.1.0")
     implementation("jakarta.activation:jakarta.activation-api:2.1.0")
     implementation("jakarta.xml.bind:jakarta.xml.bind-api:4.0.0")
-    implementation("org.apache.commons:commons-csv:1.9.0")
+    implementation("com.github.davidmoten:rtree2:0.9.3")
```

Comment:
Kauhean montaa R-puu-implementaatiota ei JVM-ympäristöille tuntunut löytyvän. Vähiten sillan alla koodatun oloiseksi valikoitui Jyrkin tiketillä mainitseman `rtree`-kirjasto, josta on käytössä saman tekijän tekemä `rtree2`. Se on muuten käytännössä identtinen alkuperäiseen verrattuna, paitsi että alkuperäinen toteuttaa tulosten haun reaktiivisena (as in RxJava). Tässä uudessa on siltä osin meidän käyttöön parempi API, ja kehittäjä lupailee, että tämä olisi se versio kirjastosta jota hän tulee pääasiallisesti ylläpitämään. 

P.S. Tuo `commons-csv` ei poistunut referenssinä, se vaan oli listalla kahteen kertaan.